src/main/kotlin/blackjack/model/Card.kt

```diff
@@ -0,0 +1,34 @@
+package blackjack.model
+
+const val ACE_VALUE_1 = 1
+const val ACE_VALUE_11 = 11
+const val ACE_VALUE_GAP = ACE_VALUE_11 - ACE_VALUE_1
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
const val ACE_VALUE_GAP = 10
```